doc/mf6io/mf6ivar/tex/gwf-lak-desc.tex

```diff
@@ -173,7 +173,7 @@
 
 \item \textcolor{blue}{\texttt{withdrawal}---real or character value that defines the maximum withdrawal rate $(L^3 T^{-1})$ for the lake. Value must be greater than or equal to zero. If the Options block includes a TIMESERIESFILE entry (see the ``Time-Variable Input'' section), values can be obtained from a time series by entering the time-series name in place of a numeric value.}
 
-\item \textcolor{blue}{\texttt{rate}---real or character value that defines the extraction rate for the lake outflow. A positive value indicates inflow and a negative value indicates outflow from the lake. RATE only applies to active (IBOUND $>$ 0) lakes. A specified RATE is only applied if COUTTYPE for the OUTLETNO is SPECIFIED. If the Options block includes a TIMESERIESFILE entry (see the ``Time-Variable Input'' section), values can be obtained from a time series by entering the time-series name in place of a numeric value. By default, the RATE for each SPECIFIED lake outlet is zero.}
+\item \textcolor{blue}{\texttt{rate}---real or character value that defines the extraction rate for the lake outflow. A positive value indicates inflow and a negative value indicates outflow from the lake. RATE only applies to outlets associated with active lakes (STATUS is ACTIVE). A specified RATE is only applied if COUTTYPE for the OUTLETNO is SPECIFIED. If the Options block includes a TIMESERIESFILE entry (see the ``Time-Variable Input'' section), values can be obtained from a time series by entering the time-series name in place of a numeric value. By default, the RATE for each SPECIFIED lake outlet is zero.}
```

Comment:
Wasn't this change already in develop?  Did you merge instead of rebase?

Reply:
@langevin-usgs Not sure what's going on.  The only intentional thing I've done at this point is create a new branch and commit to it.  It's unclear to me how or when this crept in to the PR.  Maybe my local develop is behind?  I'll try running a rebase, but I definitely have not run any merges.

Reply:
I can confirm that #1354 was merged; however, the change that it contained does not appear on the develop branch as of [66231fe](https://github.com/MODFLOW-USGS/modflow6/tree/66231febbe705b12add7c5c7eb047a248acc5c2e)

In the image below, you can see that on line 176, the change in #1354 isn't showing up?  I've never seen this before.  I guess this means the change still needs to be implemented?

![change_not_there](https://github.com/MODFLOW-USGS/modflow6/assets/3236576/9cce0063-d63e-42b7-a5d1-2c4b25400864)

Reply:
Maybe you didn't update your fork/develop with the changes in upstream/develop

Reply:
Figured out the issue here.  In #1354 I updated the .dfn file, but then failed to run mf6ivar.py, which means that the change didn't propagate out to the .tex file.  Since if ran mf6ivar.py in this workflow, part of what should have been included in #1354 is included with this PR.